app/models/post.rb

```diff
@@ -0,0 +1,20 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  validates :title, presence: true
+  validates :text, presence: true
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def update_comments_counter
+    update(comments_counter: comments.count)
+  end
```

Comment:
- Hello, I would advise you to move this method(update_comments_counter) to `comment.rb` file, because the requirement says that **Comment model should have a method that updates the comments counter for post**